### DISCUSSION TITLE
Notice: TruffleRuby now _do_ support Pattern Matching and Endless Method definitions!

### DISCUSSION THREAD

**Discussion by ParadoxV5** [ruby/prism#2697] [General]
> https://github.com/ruby/prism/blob/a0f2306d2c7245a10b0f9de464d049815da7d3df/CONTRIBUTING.md?plain=1#L13-L15

Title says it all. (:

> * Full support for the Ruby 3.2 and Ruby 3.3 syntax by adopting the [Prism](https://github.com/ruby/prism) parser, which is about twice as fast as the old parser [oracle/truffleruby#3117, oracle/truffleruby#3038, oracle/truffleruby#3039].
> * Pattern matching is now fully supported [oracle/truffleruby#3332, oracle/truffleruby#2683].
> 
> ⸺ https://github.com/oracle/truffleruby/releases/tag/graal-24.0.0, 3 weeks ago

**Comment by kddnewton** [comment]
Yay!
